fetch.bs

```diff
@@ -324,6 +330,165 @@ specialized multimap. An ordered list of key-value pairs with potentially duplic
  <li><p>Return the <a for="header">combined value</a> with <var>name</var> and <var>list</var>.
 </ol>
 
+<p>To
+<dfn export for="header list" lt="get, decode, and split|getting, decoding, and splitting" id=concept-header-list-get-decode-split>get, decode, and split</dfn>
+a <a for=header>name</a> <var>name</var> from <a for=/>header list</a> <var>list</var>, run these
+steps:
+
+<ol>
+ <li><p>Let <var>initialValue</var> be the result of <a for="header list">getting</a>
+ <var>name</var> from <var>list</var>.
+
+ <li><p>If <var>initialValue</var> is null, then return null.
+
+ <li><p>Let <var>input</var> be the result of <a>isomorphic decoding</a> <var>initialValue</var>.
+
+ <li><p>Let <var>position</var> be a <a for=string>position variable</a> for <var>input</var>,
+ initially pointing at the start of <var>input</var>.
+
+ <li><p>Let <var>values</var> be a <a for=/>list</a> of <a for=/>strings</a>, initially empty.
+
+ <li><p>Let <var>value</var> be null.
+
+ <li>
+  <p>While <var>position</var> is not past the end of <var>input</var>:
+
+  <ol>
+   <li>
+    <p>Let <var>temporaryValue</var> be the result of <a>collecting a sequence of code points</a>
+    that are not U+0022 (") or U+002C (,) from <var>input</var>, given <var>position</var>.
+
+    <p class=note><var>temporaryValue</var> might be the empty string.
+
+   <li><p>If <var>value</var> is null, then set <var>value</var> to <var>temporaryValue</var>.
+
+   <li><p>Otherwise, append <var>temporaryValue</var> to <var>value</var>.
+
+   <li>
+    <p>If <var>position</var> is not past the end of <var>input</var>, then:
+
+    <ol>
+     <li>
+      <p>If the <a for=/>code point</a> at <var>position</var> within <var>input</var> is
+      U+0022 ("), then:
+
+      <ol>
+       <li><p>Append U+0022 (") to <var>value</var>.
+
+       <li><p>Advance <var>position</var> by 1.
+
+       <li><p>Append the result of <a>collecting a sequence of code points</a> that are not
+       U+0022 (") from <var>input</var>, given <var>position</var>, to <var>value</var>.
+
+       <li>
+        <p>If <var>position</var> is not past the end of <var>input</var>, then:
+
+        <ol>
+         <li><p>Assert: the <a for=/>code point</a> at <var>position</var> within <var>input</var>
+         is U+0022 (").
```

Comment:
What about escaped quotes?

Reply:
Wow, yes, back to the drawing board. I guess I should see if this can reuse some code from the MIME type parser.